.gitattributes

```diff
@@ -0,0 +1 @@
+* text=auto
```

Comment:
It was so that people and CI checking out on windows get CRLF while unix users get LF